includes/admin/meta-boxes/class-wcs-att-meta-box-product-data.php

```diff
@@ -338,16 +358,12 @@ public static function subscription_scheme_product_content( $index, $scheme_data
 	/**
 	 * Save subscription options.
 	 *
-	 * @param  int  $post_id
+	 * @param  WC_Product  $product
 	 * @return void
 	 */
-	public static function process_product_meta( $post_id ) {
```

Comment:
I guess it's a matter of policy+process: If we agree that no public code gets moved and personal judgement is not meant to be exercised, then the code would need to stay.

My thought process here was that the benefit of reducing future deletion debt outweighs the statistically insignificant number of users potentially affected. If we have a policy for leaving code like this untouched, then we should also properly deprecate and keep track of all to-be-deleted code.

As always, extra work (deprecate) needs to be weighted against the potential harm done by taking the easy way out (hit delete). I personally follow the same cost/benefit, risk/reward thought processes with every decision I make -- a leftover from an engineering past, but I understand sometimes there must be hard rules in place for everyone, because judgement can fail.